config/joy-con-r.config.yaml

```diff
@@ -0,0 +1,9 @@
+axis_linear: 4
```

Comment:
Could you add a comment to of the controller name above this. ie `Nintendo Switch Joy-Con(R) controller`